README.md

```diff
@@ -33,6 +33,10 @@ Next, after downloading the source, copy each file into the ~/.config/ folder.
 
 ```bash
 brew install --cask font-sf-pro
+
+git clone https://github.com/kvndrsslr/sketchybar-app-font.git
```

Comment:
nit: you are missing a `cd` to the new folder. `git clone ... && cd sketchybar-app-font` would be enough